### PR TITLE
BaSP-MR-200: [Admin] (activities, trainers, subscription, classes, members) Implement react hot toast

### PR DESCRIPTION
**Pull request by lorenzomauro59**
- Change the modal alerts for toasts in Activities section (It should appear right-top of the screen)
- Change the modal alerts for toasts in Classes section (It should appear right-top of the screen)
- Change the modal alerts for toasts in all Subscriptions sections (It should appear right-top of the screen)
- Change the modal alerts for toasts in Classes section (It should appear right-top of the screen)
- Change the modal alerts for toasts in Members section (It should appear right-top of the screen)
- Remove the password input in members form (only in update)
- Remove the password input in trainers form (only in update)

### CODE REVIEW

**Comment by lorenzomauro59** [comment]
[[Admin] (activities, trainers, subs) - Implement react-hot-toast](https://trello.com/c/AD7c0Fix/200-admin-activities-trainers-subs-implement-react-hot-toast)

File: src/Components/Admin/Trainers/Form/index.jsx [hunk 1]
```diff
@@ -46,51 +42,49 @@ const AdminTrainerForm = () => {
...
-      showErrorModal(error);
+      showErrorToast(error.message);
     }
   };
 
   const updateTrainerFunction = async (id, data) => {
     try {
-      const response = await dispatch(updateTrainer(id, data));
-      showSuccesModal(response);
+      const response = await dispatch(updateTrainer(data, id));
```
Comment (fernandofernandezuy) [inline src/Components/Admin/Trainers/Form/index.jsx]: Change the position of the params in the updateTrainer function like this: updateTrainer(id, data)

**Review by fernandofernandezuy — COMMENTED** [review]
You must add "delete.firebaseUid" when you load the member or trainer in the update, in this way we can do the fetch correctly.

**Review by fernandofernandezuy — APPROVED** [review]
Great job Loren!

**Comment by araefedg** [comment]
Great work! 🚀🚀🚀

**Pull request merged**